cfme/containers/provider/openshift.py

```diff
@@ -1,15 +1,21 @@
 from cached_property import cached_property
+from os import path
 
 from . import ContainersProvider
-from cfme.utils.varmeth import variable
+
+from cfme.containers.provider import (
+    ContainersProviderDefaultEndpoint, ContainersProviderEndpointsForm
+)
+
```

Comment:
drop these newlines, single block for local imports

Reply:
:+1: